github/github.go

```diff
@@ -0,0 +1,663 @@
+/*
```

Comment:
So the github UI still isn't showing it as a mv, but `git blame` works, and if you look at the commits you can see the "specific" changes.